pkg/cli/cvd.go

```diff
@@ -412,3 +450,67 @@ func verifyCVDHostPackageTar(dir string) error {
 	}
 	return nil
 }
+
+type LocalArtifactsSrcs struct {
+	Images            map[string]string
+	Bootloader        string
+	CVDHostPackageTar string
+}
+
+func (s *LocalArtifactsSrcs) validate() error {
+	if s.Bootloader == "" {
+		return errors.New("missing bootloader source")
+	}
+	if s.CVDHostPackageTar == "" {
+		return errors.New("missing cvd host package tar source")
+	}
+
+	// TODO: Make it configurable.
+	var allowedImageKeys = []string{
+		"boot",
+		"init_boot",
+		"super",
+		"userdata",
+		"vbmeta",
+		"vbmeta_system",
+		"vbmeta_vendor_dlkm",
+		"vbmeta_system_dlkm",
+		"vendor_boot",
+	}
+	for k := range s.Images {
+		if !contains(allowedImageKeys, k) {
+			return fmt.Errorf("invalid image key: %q", k)
+		}
+	}
+	return nil
+}
+
+func (s *LocalArtifactsSrcs) names() []string {
+	result := []string{s.Bootloader, s.CVDHostPackageTar}
+	for _, v := range s.Images {
+		result = append(result, v)
+	}
+	return result
+}
+
+func loadLocalArtifacsSrcs(path string) (*LocalArtifactsSrcs, error) {
+	b, err := os.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+	decoder := toml.NewDecoder(bytes.NewReader(b))
+	r := &LocalArtifactsSrcs{}
```

Comment:
IIUC, toml contains metadata of CF images & cvd-host_package.tar.gz. Could we have example toml file so that users can make files by themselves?

Reply:
good point, rephrased the flag documentation adding the example there.

Reply:
:+1: